### DISCUSSION TITLE
Incentivise An Exchange to Service the Local Community

### DISCUSSION THREAD

**Discussion by skynet-git** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#36] [General]
I would like to offer a suggestion... 

The virtual asset sector is a global sector and licensing/registration is meant for businesses who are based out of or operating from the Cayman Islands. This typically applies to larger businesses who are already established and who typically operate in large markets.

Cayman is a small market – we have a population of approx 65,000 persons and only a small fraction of these would actively engage in virtual asset trades, like buying or selling cryptoassets. This means our local market is typically too small for the larger companies who are established here and therefore they may not allow local Caymanians and residents to use their services.

I think this sector is the future, and it’s imperative that young Caymanians are exposed to these new technologies in a safe manner – but how can they do that if there is no exchange serving the Cayman market?

I would suggest the following to help in this regards: Offer a discount on the CIMA licensing fees IF the applicant serves the local market. The discount should be sufficiently enticing to entice a larger VASP to register a local company (e.g. not an exempt company).

In this manner, an exchange who wishes to operate from here could save considerable sums by servicing the local market, AND caymanians benefit because they have access to crypto markets through an entity licensed with CIMA (instead of having to use VPNs or use foreign VASPs as they have to do now).

What does the community think of this idea?

**Comment by FinalTwist** [comment]
That's a great suggestion - I hate it how the local community needs to seek out exchanges outside the country to trade.   Would the lawyers in the room here be able to suggest to the Ministry what type of fee discount, or other incentive might be enough to incentivize VASPs to set up a local company?

**Comment by AcrobaticPensioner** [comment]
I also like this suggestion although I am not sure the incentives provided to company in the form discounted licensing fees are likely to be worthwhile trade for the CIG for the relatively small number of people  who'd be eligible to access their services. Why not make it a condition of VASP licensing that Caymanian nationals, businesses and residents are not unfairly disadvantaged by license holders.

**Reply by FinalTwist** [reply]
The issue here is not with the VASP Act, I'm sure all registered VASPs would be happy to service the local market.  The issue lies with the Caymanian/Exempt company regimes.  Most companies come here and don't want to set up with a 60% Caymanian ownership in order to offer services locally so they set up as an exempt company and/or in an economic zone.  That means that by default they can't operate locally unless they apply for an LCCL.  So in order to offer services locally they would need to apply for an LCCL which is a process that costs, etc.

I don't think there is any way for a VASP to offer services locally, unless I am mistaken?
